BangazonAPI/Models/Department.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Department
+    {
+        public int id { get; set; }
+        public string name { get; set; }
+        public int budget { get; set; }
+        List<Employee> departmentEmployees = new List<Employee>
```

Comment:
The name of this list will be exactly what the end user sees when they get the response back from the API-- let's name this list something more descriptive and user-friendly. I'd recommend `Employees` or `employees`, since they'll already know they're looking at employees that are associated with a department.

Reply:
Also, be sure to add a getter and setter for your list of employees 👍